workflow-templates/php-library-static.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "DVSA Organization Workflow - PHP Static Analyser",
+    "description": "DVSA Organization PHP static analyser starter workflow. This workflow runs PHP static analysis. See the README for running instruction and configuration options.",
+    "categories": [
+        "php"
```

Comment:
Really minor, but I believe this will cause issues while trying to filter actions by categories, which is going to be useful once we start adding more.

`PHP` rather than `php` per the spec: https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml#L4863